lib/authenticated-api/authenticated-api-function-props.ts

```diff
@@ -11,4 +11,5 @@ export interface AuthenticatedApiFunctionProps {
   vpcSubnets?: ec2.SubnetSelection;
   securityGroups?: Array<ec2.ISecurityGroup>;
   memorySize?: number;
+  bundling?: BundlingOptions;
```

Comment:
Is this type correct? I've just had a quick look at the [docs](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.BundlingOptions.html) and was expecting to see something related to `externalModules` & `nodeModules`.

Reply:
comment: Good catch. I had imported the wrong thing.